src/python/pants/backend/python/goals/run_pex_binary_integration_test.py

```diff
@@ -13,10 +13,7 @@
 
 use_deprecated_semantics_args = pytest.mark.parametrize(
```

Comment:
This and it's usage can be removed altogether.

I'll push to this branch next opportunity if someone else doesn't beat me to it.